src/vsr/replica.zig

```diff
@@ -1600,13 +1600,15 @@ pub fn ReplicaType(
                 return;
             }
 
-            if (self.status != .normal) {
-                log.debug("{}: on_prepare: ignoring ({})", .{ self.replica, self.status });
+            if (message.header.view < self.view) {
+                log.debug("{}: on_prepare: ignoring (older view)", .{self.replica});
                 return;
             }
```

Comment:
Hm, I'd still just make `is_repair` to return `true` for this use case, this _should_ work and would lead to simpler code, no?

Reply:
Oops, I hadn't _completely_ understood your suggestion earlier then. Indeed it would simplify things, let me do this!

Reply:
Yeah, I formulated it in a quiet confusing way, sorry about that! It's much easier to think than to communicate :rofl: